Objects/obmalloc.c

```diff
@@ -467,6 +468,7 @@ _PyMem_ArenaAlloc(void *Py_UNUSED(ctx), size_t size)
     if (ptr == MAP_FAILED)
         return NULL;
     assert(ptr != NULL);
+    _PyAnnotateMemoryMap(ptr, size, "cpython:obmalloc");
```

Comment:
obmalloc is the filename, but the memory allocator is called pymalloc: https://docs.python.org/dev/c-api/memory.html#default-memory-allocators

```suggestion
    _PyAnnotateMemoryMap(ptr, size, "cpython:pymalloc");
```

Reply:
Let's be consistent, other annotation follows filename.